app/views/layouts/_search.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="d-flex justify-content-between" style="margin-top:30px">
+  <div>
+    <form class="project_search project-s" id="project_search project-s" action="/projects" accept-charset="UTF-8" method="get">
+      <input id="search-project" placeholder="Search a project" type="text" name="q[name_cont]" id="q_name_cont">
+    </form>
```

Comment:
utilise toujours les helpers Rails `form_for` etc

Reply:
noté, prochaine pr